dotcom-rendering/configs/webpack/server.dev.mjs

```diff
@@ -10,18 +10,114 @@
 // THIS IS ONE IS TRICKIER, THIS IS MORE OF A GUIDE FOR HOW TO PROCEED.
 // IT'S NOT FINISHED.
 
+import { resolve } from 'node:path';
+import { fileURLToPath } from 'node:url';
+import bodyParser from 'body-parser';
+// eslint-disable-next-line import/no-named-as-default -- this is the Webpack way
+import webpack from 'webpack';
+import webpackHotServerMiddleware from 'webpack-hot-server-middleware';
 import { mergeWithRules } from 'webpack-merge';
 import nodeExternals from 'webpack-node-externals';
+import { success } from '../../../scripts/log.js';
+import { getContentFromURLMiddleware } from '../../src/server/lib/get-content-from-url.js';
 import { base } from './base.mjs';
 import { server } from './server.mjs';
 
+const DIRNAME = fileURLToPath(new URL('.', import.meta.url));
+
 /** @type {import("webpack").Configuration} */
-export const devServer = {
+export const serverDev = {
+	devServer: {
+		compress: false,
+		hot: false,
+		liveReload: true,
+		client: {
+			logging: 'warn',
+			overlay: true,
+		},
+		port: 3030,
```

Comment:
Could this be made a const to make it easier to spot and change if required?

Reply:
Possibly, although the benefit of these webpack configs is that they are shorter and only have meaningful fields. If we extract each of the fields to const, then we are no further along in achieving prominence for any specific one.

We could move it to the top of the object if we feel that it’s more important?